### ISSUE TITLE
Can't access phpmyadmin after manually installing on a fresh server environment

### ISSUE THREAD

**Issue by Benqzq** [phpmyadmin/phpmyadmin#13250]
On an Ubuntu 16.04.2 VPS with an Apache2 server environment I installed PHPmyadmin, as I always do:

    (
    cd /var/www/html 
    sudo wget https://www.phpmyadmin.net/downloads/phpMyAdmin-latest-all-languages.zip
    sudo find ./ -type f -name '*phpMyAdmin*.zip' -exec unzip {} \;
    sudo find ./ -type d -name 'phpMyAdmin-*' -exec mv {} phpmyadmin \;
    )

In my Browser I did `VPS_IP/phpmyadmin`. I then got:

> This page isn’t working
> 
> VPS_IP is currently unable to handle this request.
>
>HTTP ERROR 500

This is wired because a Wordpress website on this server **works as usual** - I can access its homepage and all other pages and change it, so it seems the Apache server is fine:

* no errors in `auth.log` or `error.log`.
* No relevant warnings in ctl.

I also ruled out a phpmyadmin iptables block because this 500 internal error happens in computers from around the globe.

---

I went after many solutions dealing with 404 in PHPmyadmin but all of these deal with the `apt-get install` method and didn't help me.

Please, do you have any idea what could went wrong?

  [1]: https://i.stack.imgur.com/1oY85.png

**Comment by Benqzq** [comment]
Interesting: If I move phpmyadmin from /var/www/html to /var/www/html/website-that-works, it does work and I can access it from the browser without error, but when it's in document root, I just can't...

**Comment by Benqzq** [comment]
This seems to be associated with a malconfigured Varnish.

When I did nano /etc/apache2/sites-available and changed, changed 8080 to 80, saved and restart the apache2 service, the problem was solved.

I changed it to 8080 when I tried to install Varnish but I might have malconfigured it on the way.

[Keep following here](https://unix.stackexchange.com/questions/363433/phpmyadmin-inaccessible-on-document-root-but-accessible-on-any-directory-inside).
